src/Sarif.FunctionalTests/v2/SpecExamples/InlineExternalProperties.sarif

```diff
@@ -0,0 +1,561 @@
+                                    },
+                                    "replacements": [
+                                        {
+                                            "deletedRegion": {
+                                                "byteOffset": 109
+                                            },
+                                            "insertedContent": {
+                                                "binary": "PSBudWxs"
+                                            }
+                                        }
+                                    ]
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ]
+        }
+    ],
+    "runs": [
+        {
+            "tool": {
+                "name": "CodeScanner",
+                "semanticVersion": "2.1.0",
+                "driver": {
+                    "name": "CodeScanner Legacy File Format Converter",
+                    "fullName": "CodeScanner Converter Tool",
+                    "version": "4.3",
+                    "semanticVersion": "4.3.0",
+                    "dottedQuadFileVersion": "4.3.0.0"
+                }
+            },
+            "results": [],
+            "columnKind": "utf16CodeUnits",
+            "externalPropertyFiles": {
+                "conversion": {
+                    "location": {
+                        "uri": "sarif:/inlineExternalProperties/0"
+                    }
+                },
+                "graphs": {
+                    "location": {
+                        "uri": "sarif:/inlineExternalProperties/0"
+                    }
+                },
+                "externalizedProperties": {
+                    "location": {
+                        "uri": "sarif:/inlineExternalProperties/0"
+                    }
+                },
+                "artifacts": [
+                    {
+                        "location": {
+                            "uri": "sarif:/inlineExternalProperties/0"
+                        }
+                    },
+                    {
+                        "location": {
+                            "uri": "sarif:/inlineExternalProperties/1"
+                        }
+                    }
+                ],
+                "invocations": [
+                    {
+                        "location": {
+                            "uri": "sarif:/inlineExternalProperties/0"
+                        }
+                    },
+                    {
+                        "location": {
+                            "uri": "sarif:/inlineExternalProperties/1"
+                        }
+                    }
+                ],
+                "logicalLocations": [
+                    {
+                        "location": {
+                            "uri": "sarif:/inlineExternalProperties/0"
+                        }
```

Comment:
ok, while creating this file, i was thinking if all these properties really need to be an array of location?? the item at inlineExternalProperties.logicalLocations is an array already which this references. thoughts?

Reply:
clarified offline, this is by design. we can have multiple chunks spread across different externalProperties objects.

---
In reply to: [265397024](https://github.com/Microsoft/sarif-sdk/pull/1342#discussion_r265397024) [](ancestors = 265397024)